images/tags.md

```diff
@@ -90,3 +99,36 @@ updates:
   associated with a specific project's major version. You can apply the `:v1`
   tag to the most recent build for the image, while you can use `:v1.3` or
   `:v1.3.1` to pull a more specific tag version.
+
+## Coder recommendation
```

Comment:
This whole section is a strawman that is intended to express the same thing as the "Latest versus numbered version" section... just that they should be deliberate and that it's flexible but also moving stuff around in unintuitive ways can slow the team or make excessive tag management.